### PR TITLE
Add support for PHP 8.2 to composer.json

### PR DESCRIPTION
**Pull request by TimWolla**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | yes
| RFC           | no
| QA            | no

### Description

Dependency for laminas/laminas-progressbar#17.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
         }
     },
     "require": {
-        "php": "^7.4 || ~8.0.0 || ~8.1.0",
+        "php": "^7.4 || ~8.0.0 || ~8.1.0 || ~8.2.0",
```
Comment (Ocramius) [inline composer.json:35]: Hmm, I think the various `require-dev` will block you here. 

See https://github.com/laminas/laminas-view/blob/69ea122cd53f7839e58cb250975932332e542524/.laminas-ci.json#L7-L10 for an example of how to silence this temporarily.

**Review by Ocramius — CHANGES_REQUESTED** [review]

**Comment by Ocramius** [comment]
Ooof, this will be a pain:

```
E: Unable to locate package php8.2-mongodb
E: Couldn't find any package by glob 'php8.2-mongodb'
```

@boesing WDYT? Can we just skip MongoDB tests, and remove the extension from CI? It's such a massive PITA.

**Comment by boesing** [comment]
should work skipping these tests as of now. 
why does this even need mongo? can we change to apcu probably?

**Comment by boesing** [comment]
Main Problem is that we have fails again - I would prefer having missing packages do not break CI. Tests have to fail when extensions are not available - not container setup. maybe we can modify the container somehow as well.

when adding 8.1, I used shivammathur php-builder which handled plenty of extensions itself. and if not available we just skipped Installation. now this whole setup fails 😔

**Comment by Ocramius** [comment]
Agreed, but we can't really be the integrator for MongoDB when it is such a niche use-case :| 

Having stability for the main use-case is kinda more important here, even if MongoDB then shits the bed

**Comment by Ocramius** [comment]
Still no package released: I really think we should ignore mongodb here, perhaps even deprecate it :thinking:

**Comment by TimWolla** [comment]
FWIW: The laminas-progressbar update was no longer blocked on this and the removal of MongoDB is more than I'm comfortable doing as a contributor. Thus I don't plan to pursue this further.

Feel free to cherry-pick the https://github.com/laminas/laminas-session/commit/479fe973b7dbabad0fbc5cf25240439532d0c4dd.patch commit into 2.14.x (it appears to fail Psalm, though). The remaining changes are trivial anyway.

**Comment by Ocramius** [comment]
Handled by #63 meanwhile. Thanks nonetheless, @TimWolla!

**Pull request closed without merging**